index.js

```diff
@@ -1,8 +1,37 @@
+const dark=document.querySelector(".dark");
 var slide = document.querySelector("#slider");
 var btn1 = document.querySelector("#btn1");
 var btn2 = document.querySelector("#btn2");
 var btn3 = document.querySelector("#btn3");
 
+if(localStorage.getItem("dark")==="dark-theme"){
+    document.body.classList.add("dark-theme")
+    dark.classList.remove("fa-sun")
+    dark.classList.add("fa-moon")
+}
+else{
+    document.body.classList.add("light-theme")
+}
+
+dark.addEventListener('click',()=>{
+    if(dark.classList.contains("fa-sun")){
```

Comment:
You can add toggle method to toggle class

Reply:
I will check that and try to add that now if possible or-else I will do that in future